core/domain/email_manager.py

```diff
@@ -300,10 +298,10 @@ def send_role_notification_email(
     """
     # Editor role email body and email subject templates.
     # pylint: disable=invalid-name
-    editor_role_email_subject_template = (
+    EDITOR_ROLE_EMAIL_SUBJECT_TEMPLATE = (
```

Comment:
Actually -- sorry, the point of my comments here previously was to get rid of the "disable=invalid-name". However, I realize now that the issue is mainly that the name is too long, and it's not an issue with casing. I think I might have been mistaken about all-caps, too -- not sure if pylint allows that within a function, so feel free to change it back to lowercase if needed.

To get rid of the pylint pragma, how about email_subject_template and email_body_template? Since this is in a function related to role notification emails, the additional qualifiers seem unnecessary.

Thanks, and sorry for the mistake.